src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -1775,14 +1914,18 @@ class IrsPlan extends React.Component<
 
     const columns = [
       {
-        Header: () => (
-          <Input
-            checked={planJurisdictionIds.length === filteredJurisdictions.length}
-            className="plan-jurisdiction-select-all-checkbox"
-            onChange={onToggleAllCheckboxChange}
-            type="checkbox"
-          />
-        ),
+        Header: () => {
+          const isChecked = this.getIsJurisdictionPartiallySelected(this.state.focusJurisdictionId);
```

Comment:
This line right here is consuming tons of browser resources, simplifying it to use what we already have in state instead of re-calculating if the administrative jurisdiction has any selected descendant operational jurisdictions.